File: data/code_pairs/fortran/1901.f90
module sum_module
contains
  function compute_sum(n) result(sum)
    implicit none
    integer, intent(in) :: n
    integer :: sum
    integer :: i

    sum = 0
    do i = 1, n
       sum = sum + i
    end do
  end function compute_sum
end module sum_module

File: data/code_pairs/cpp/1901.cpp
#include <iostream>

int compute_sum(int n) {
    int sum = 0;
    for (int i = 1; i <= n; ++i) {
        sum += i;
    }
    return sum;
}

int main() {
    std::cout << "The sum is " << compute_sum(10) << std::endl;
    return 0;
}